test/extensions/filters/http/jwt_authn/filter_integration_test.cc

```diff
@@ -23,7 +23,7 @@ namespace JwtAuthn {
 namespace {
 
 const char HeaderToFilterStateFilterName[] = "envoy.filters.http.header_to_filter_state_for_test";
-
+const Http::LowerCaseString WwwAuthenticateHeader("www-authenticate");
```

Comment:
This will lead static-initialization fiasco hence flaky test, use a function with CONSTRUCT_ON_FIRST_USE.

Reply:
Swapped it over to the same value we use in the filter itself